openrr-apps/src/robot_teleop_config.rs

```diff
@@ -56,6 +59,9 @@ pub struct RobotTeleopConfig {
     pub control_nodes_config: ControlNodesConfig,
     #[serde(default)]
     pub gil_gamepad_config: GilGamepadConfig,
+    #[cfg(feature = "ros")]
+    #[serde(default)]
+    pub joy_gamepad_config: JoyGamepadConfig,
```

Comment:
I feel that it's better to use `RosJoyGamepadConfig`.
Someone may feel that `joy` means ROS actually, but it's not so usual.

Reply:
#721

Reply:
@OTL Should I rename the file as well?